cg/server/dto/samples/samples_response.py

```diff
@@ -1,19 +1,66 @@
+from datetime import datetime
 from pydantic import BaseModel
 
 from cg.constants.subject import Sex
 
 
+class ApplicationVersionDTO(BaseModel):
+    id: int
+    version: int | None = None
+    valid_from: datetime | None = None
+    price_standard: int | None = None
+    price_priority: int | None = None
+    price_express: int | None = None
+    price_research: int | None = None
+    price_clinical_trials: int | None = None
+    comment: str | None = None
+    created_at: datetime | None = None
+    updated_at: datetime | None = None
+    application_id: int | None = None
+
+
+class ApplicationDTO(BaseModel):
+    id: int
+    tag: str | None = None
+    prep_category: str | None = None
+    is_external: bool | None = None
+    description: str | None = None
+    is_accredited: bool | None = None
+    turnaround_time: int | None = None
+    minimum_order: int | None = None
+    sequencing_depth: int | None
+    min_sequencing_depth: int | None = None
+    target_reads: int | None = None
+    percent_reads_guaranteed: int | None = None
+    sample_amount: int | None = None
+    sample_volume: str | None = None
+    sample_concentration: str | None = None
+    sample_concentration_minimum: float | None = None
+    sample_concentration_maximum: float | None = None
+    sample_concentration_minimum_cfdna: float | None = None
+    sample_concentration_maximum_cfdna: float | None = None
+    priority_processing: bool | None = None
+    details: str | None = None
+    limitations: str | None = None
+    percent_kth: int | None = None
+    comment: str | None = None
+    is_archived: bool | None = None
+    created_at: datetime | None = None
+    updated_at: datetime | None = None
+
+
 class CustomerDto(BaseModel):
     internal_id: str
     name: str
```

Comment:
We returned a lot more customer data before as well

Reply:
It is not used for anything though so there is no need to return it